### ISSUE TITLE
Kendo combox does not expose API to add aria-labelledby

### ISSUE THREAD

**Issue by ylux587** [telerik/kendo-ui-core#5899]
### Reproduction of the problem
In our application we use combox and wrap it as a react component. However, we found we cannot add "aria-labelledby" attribute. Seems no API to support this attribute to add.

### Current behavior
No aria-labelledby generated in html and no api to set this attribute

### Expected/desired behavior
An API to set this attribute (aria-labelledby)

### Environment

* **Kendo UI version:** 2019.3.1114
* **jQuery version:** 3.3.28
* **Browser:** Chrome 81.0.4044.122

**Comment by GaloisGirl** [comment]
You have the [input](https://docs.telerik.com/kendo-ui/api/javascript/ui/combobox/fields/input) property:

```
$("#combobox").kendoComboBox();
var combobox = $("#combobox").data("kendoComboBox");
var input = combobox.input;
input[0].setAttribute("aria-label", "your label here");  
```

You can also set an id or label on the `input` element before creating the CombBox.

**Comment by XingyangGeng** [comment]
HI @GaloisGirl :
   We are using ComboBoxOptions like:
 <ComboBox
       dataSource=""
       dataTextField=""
       dataValueField=""
  </ComboBox>
In this case, how can we add the "aria-label" property?

**Comment by GaloisGirl** [comment]
@XingyangGeng isn't that React? Shouldn't you file the ticket here? https://github.com/telerik/kendo-react/issues

**Comment by XingyangGeng** [comment]
Hi @GaloisGirl ,
More specifically, we are using Kendo UI for jQuery widgets wrapped as React components.
```
    kendoMultiSelect(): JQuery;
    kendoMultiSelect(options: kendo.ui.MultiSelectOptions): JQuery;
    data(key: "kendoMultiSelect"): kendo.ui.MultiSelect;
```
https://github.com/kendo-labs/k-ui-react-jquery-wrappers

**Comment by Dimitar-Goshev** [comment]
Kendo UI ComboBox automatically creates an area-labelledby attribute when a label element is associated with it, as shown on our [ComboBox demo](https://demos.telerik.com/kendo-ui/combobox/index). Adding a label is one way to have the attribute in place. Another approach is to take reference to the ComboBox after being initialized and then set its attribute manually as suggested above.

As a side note, please have in mind that the referenced react wrappers repository is not supported by us and its latest commit is 4 years old. Thus, we cannot provide support for its integration with Kendo UI.
